packages/core/src/NodeExecuteFunctions.ts

```diff
@@ -2064,8 +2072,6 @@ const getBinaryHelperFunctions = ({
 		prepareBinaryData(binaryData, executionId!, filePath, mimeType),
 	setBinaryDataBuffer: async (data, binaryData) =>
 		setBinaryDataBuffer(data, binaryData, executionId!),
-	copyBinaryFile: async (filePath, fileName, mimeType) =>
```

Comment:
I added this function very recently. it's very unlikely that anyone besides us is using this in their nodes.

Reply:
I know, I saw it was added in November, my idea is just to be extra safe in case someone ever tried using it. It doesn't harm to add a error throw explaining how to proceed, especially since it was visible in the `Code` node and people could have tried using it.

Reply:
Yes, that looks good! Thanks 💯